src/styles/product-card.scss

```diff
@@ -0,0 +1,80 @@
+.product-card {
+  border: 1px solid #fff;
```

Comment:
The color value for the border should be a variable instead of a hardcoded value to maintain consistency and ease of maintenance. Consider using a variable like $border-color.